tests/load/c/ginkgo_test.go

```diff
@@ -53,21 +63,35 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 })
 
 var _ = ginkgo.Describe("[Load Simulator]", ginkgo.Ordered, func() {
-	var network *tmpnet.Network
+	var (
+		network  *tmpnet.Network
+		tracker  *load.Tracker[common.Hash]
```

Comment:
Is it possible to have a tracker per subtest? Sharing the tracker between runs might not be a great idea given the orchestrator uses it to re-configure itself.
What's wrong with having a prometheus server start & stop per subtest? Especially given the last test would exit the server as a subtest would 🤔 Unless I'm missing something, what we could do is have the execute function sleep a few seconds on termination (context aware, log it out when sleeping), so it can be scrapped one last time? The sleep duration should ideally be the scrape period for the Prometheus load target